src/server/claimQueue/claimQueueAPI.js

```diff
@@ -46,21 +46,31 @@ const ClaimQueue = {
   },
 
   async enqueue(user, storage, log) {
-    log.debug('claimqueue:', { allowed: conf.claimQueueAllowe, queue: user.claimQueue })
-    //if queue not enabled, user already in queue or user already whitelisted we skip adding to queue
-    if (conf.claimQueueAllowed <= 0 || user.claimQueue) {
-      return { ok: 0, queue: user.claimQueue || { status: 'verified' } }
+    const { claimQueueAllowed } = conf
+    const { claimQueue } = user
+
+    log.debug('claimqueue:', { allowed: claimQueueAllowed, queue: claimQueue })
+
+    // if user already in queue or user already whitelisted we skip adding to queue
+    if (claimQueue) {
+      return { ok: 0, queue: claimQueue }
     }
-    const totalQueued = await storage.model.count({ 'claimQueue.status': { $exists: true } })
-    const openSpaces = conf.claimQueueAllowed - totalQueued
 
+    const totalQueued = await storage.model.count({ 'claimQueue.status': { $exists: true } })
+    const openSpaces = claimQueueAllowed - totalQueued
     let status = openSpaces > 0 ? 'approved' : 'pending'
-    //if user was added to queue tag him in mautic
-    if (['test', 'development'].includes(conf.env) === false && user.mauticId && status === 'pending')
-      Mautic.addContactsToSegment([user.mauticId], conf.mauticClaimQueueSegment).catch(e => {
+
+    // if user was added to queue tag him in mautic
+    if (!['test', 'development'].includes(conf.env) && user.mauticId && status === 'pending') {
+      try {
+        await Mautic.addContactsToSegment([user.mauticId], conf.mauticClaimQueueSegment)
```

Comment:
mauticClaimQueueSegment -> mauticClaimQueueSegmentId

Reply:
@yaroslav-fedyshyn-nordwhale please use `+/-` (Suggest) for one-line fixes. It's easy to create fix commit then , directly from web UI